### ISSUE TITLE
WCT link hardwired into all ncISO records is broken

### ISSUE THREAD

**Issue by rsignell-usgs** [Unidata/threddsIso#36]
The ncISO code [hardwires broken viewer links for some WCT beta service into the ISO records](https://github.com/Unidata/threddsIso/blob/master/src/main/resources/xsl/nciso/UnidataDD2MI.xsl#L977), like in this ISO record:

http://thredds.ucar.edu/thredds/iso/grib/FNMOC/WW3/Global_1p0deg/Best?catalog=http%3A%2F%2Fthredds.ucar.edu%2Fthredds%2Fcatalog%2Fgrib%2FFNMOC%2FWW3%2FGlobal_1p0deg%2Fcatalog.html&dataset=grib%2FFNMOC%2FWW3%2FGlobal_1p0deg%2FBest

which has a WCT viewer link of:
http://www.ncdc.noaa.gov/oa/wct/wct-jnlp-beta.php?singlefile=http://thredds.ucar.edu/thredds/dodsC/grib/FNMOC/WW3/Global_1p0deg/Best
https://www.ncdc.noaa.gov/oa/wct/wct-jnlp-beta.php

which gives a 404.

Is there an updated WCT endpoint that would work?

Would it not be good to provide the godiva endpoint in the ISO as a viewer endpoint, which *does* work, and which actually is provided by THREDDS?

http://thredds.ucar.edu/thredds/godiva2/godiva2.html?server=http://thredds.ucar.edu/thredds/wms/grib/FNMOC/WW3/Global_1p0deg/Best

**Comment by lesserwhirls** [comment]
Interesting, I don't think I've ever noticed the wct viewer link in there before. @noaaroland - do you know of an updated url for wct?

The thing about setting godiva as the viewer is that wms isn't enabled by default on 4.x servers, and most servers I've visited do not turn it on.

**Comment by rsignell-usgs** [comment]
@lesserwhirls, I should have been more clear:  I'm not proposing to hardwire godiva in there (I think hardwiring **anything** in the nciso code is a bad idea), but only adding it if WMS is enabled.   nciso has the information it needs to do that already!

**Comment by lesserwhirls** [comment]
Ah, yes, that makes sense. I *think* that could be added pretty easily. However, I'm a bit reluctant to do that asap, as I would like to see the two versions of the iso code merged so that we have one codebase, if possible (that is, [this](https://github.com/Unidata/threddsIso/issues/7#issuecomment-317129244)). @noaaroland @kevin-obrien - where does the standalone app stand at this point? Is there still interest in getting things under one codebase?

**Comment by kevin-obrien** [comment]
Yes, @lesserwhirls  absolutely under once codebase would be ideal. At the NetCDF meeting in Sept, you mentioned that you thought there were a few libraries and/or methods missing.  We should follow up on this discussion soon and figure out how to identify what is missing...
